en/docs/references/ballerina-expression-syntax.md

```diff
@@ -1,422 +1,495 @@
 
-## Supported Literals 
+## Variables
+A variable declaration statement declares a value of a specific type (`string`, `integer`, etc.).  You can use a previously declared variable at a later phase in the logical process of your service or integration.
 
-### String
+### int
 
-A string is simply a sequence of characters. It is required to use double quotes (`"`)  to mark the boundary between the starting and ending characters.
+The `int` type is 64-bit signed integers (same as long in Java). A signed integer is an integer with the  `+` or `-` character in the front. However, You can also define a positive integer or zero using only numerical characters. The `int` type supports the usual [arithmetic operators: + - / %](#arithmetic-operators). The operator precedence is the same as C. Integer overflow is a runtime error in Ballerina. 
 
-#### Examples
+###### Examples:
 
-  ```ballerina
-  "Colombo"
-  ```
-  ```ballerina
-  "12345"
-  ```
-  ```ballerina
-  "Ballerina is a programming language intended for network-distributed applications."
-  ```
-
-### Signed Integer
-
-A signed integer is an integer with the  `+` or `-` character in the front, indicating the sign. You can also define a positive integer or zero using only numerical characters without the sign. Using the signed integer type, you can give positive integers, negative integers, and zero as an input to an expression-supported field.
-
-#### Examples
+To initialize a variable of type `int` in Choreo, you can use one of the following expressions:
 
   ```ballerina
   1234
-  ```
-  ```ballerina
+  
   -10
-  ```
-  ```ballerina
+  
   +2020
   ```
 
-### Decimal Floating-Point Number
+### float
+
+The `float` type is IEEE 64-bit binary floating-point (same as double in Java) and supports the same arithmetic operators as `int`. You have the option to use `+` or `-` characters in the front to indicate the sign.
 
-A decimal floating-point number is a number with a decimal point. You have the option to use `+` or `-` characters in the front to indicate the sign.
+###### Examples:
 
-#### Examples
+To initialize a variable of type `float` in Choreo, you can use one of the following expressions:
 
   ```ballerina
   0.3353
-  ```
-  ```ballerina
-  2044.24
-  ```
-  ```ballerina
+  
   -8593.992
-  ```
-  ```ballerina
+  
   +950.930
   ```
-  ```ballerina
-  0
-  ```
 
-### Boolean 
+### boolean
 
 The boolean data type has one of the two possible values, `true` or `false`.
 
-#### Examples
+### string
 
+The `string` type represents an immutable sequence of zero or more Unicode characters. In Ballerina, there is no character type. Instead, a string of length is one represents a character. Two string values are considered equal (==) if both sequences have the same characters. You can use <, <=, >, and >= operators on string values. They work by comparing code points. Unpaired surrogates are not allowed.
```

Comment:
"Instead, a string of length is one represents a character" is unclear and sounds grammatically incorrect.

Reply:
Fixed.